big_bang.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+set -e 
+
+pkg_name=$1
+echo "
+
+ ▄▄▄▄    ██▓  ▄████     ▄▄▄▄    ▄▄▄       ███▄    █   ▄████ 
+▓█████▄ ▓██▒ ██▒ ▀█▒   ▓█████▄ ▒████▄     ██ ▀█   █  ██▒ ▀█▒
+▒██▒ ▄██▒██▒▒██░▄▄▄░   ▒██▒ ▄██▒██  ▀█▄  ▓██  ▀█ ██▒▒██░▄▄▄░
+▒██░█▀  ░██░░▓█  ██▓   ▒██░█▀  ░██▄▄▄▄██ ▓██▒  ▐▌██▒░▓█  ██▓
+░▓█  ▀█▓░██░░▒▓███▀▒   ░▓█  ▀█▓ ▓█   ▓██▒▒██░   ▓██░░▒▓███▀▒
+░▒▓███▀▒░▓   ░▒   ▒    ░▒▓███▀▒ ▒▒   ▓▒█░░ ▒░   ▒ ▒  ░▒   ▒ 
+▒░▒   ░  ▒ ░  ░   ░    ▒░▒   ░   ▒   ▒▒ ░░ ░░   ░ ▒░  ░   ░ 
+ ░    ░  ▒ ░░ ░   ░     ░    ░   ░   ▒      ░   ░ ░ ░ ░   ░ 
+ ░       ░        ░     ░            ░  ░         ░       ░ 
+      ░                      ░                              
+
+"
+
+echo "Spawning ${pkg_name}_description directory..."
+
+catkin_create_pkg ${pkg_name}_description rospy
+cd ${pkg_name}_description/
+mkdir launch meshes rviz urdf
+
+cd urdf/
+echo
+echo "Enter config details : "
+echo
+
+read -p 'documentId : ' doc_id
```

Comment:
This interface doesn't seem to hide any complexity so, if I were the maintainer, I would have some concerns about code bloat.